website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -342,7 +342,7 @@ Finally, run your bundle. If your bundle image is hosted in a registry that is p
 has a custom CA, these [configuration steps][image-reg-config] must be complete.
 
 ```sh
-operator-sdk run bundle example.com/memcached-operator-bundle:v0.0.1
+operator-sdk run bundle <docker registry host>/example.com/memcached-operator-bundle:v0.0.1
```

Comment:
```suggestion
operator-sdk run bundle <some-registry>/memcached-operator-bundle:v0.0.1
```
Not necessary docker. That can be any registry. 

Only this nit to address. Otherwise, LGTM. 

Feel free to ping me into the slack for we get it merged.

Reply:
Thanks for review.  Right.  It can be any image registry.  It needs the ip address, domain, name of the image and tag.  So 

```  
<image registry host>/example.com/memcached-operator-bundle:v0.0.1
```
is better.  The `example.com/memcached-operator-bundle:v0.0.1` is for consistency from the previous steps.

Reply:
I would prefer as we do here: https://book.kubebuilder.io/quick-start.html and in another places.

<some registry> is == e.g. quay.io/organization/

Reply:
OK. It's better enough to users.  Thanks!